malaria-app-android/src/main/java/com/peacecorps/malaria/notification/DrugReminderNotification.java

```diff
@@ -0,0 +1,107 @@
+        NotificationManager notificationManager;
+        notificationManager = (NotificationManager) context
+                .getSystemService(Context.NOTIFICATION_SERVICE);
+
+        PendingIntent contentIntent = PendingIntent.getActivity(context, 0,
+                new Intent(context, MainActivity.class), 0);
+
+
+       /* Add Actions in Notification
+        /*Action Not Taken*/
+        Intent intentNotTaken = new Intent(context, DrugReminderReceiver.class);
+       /* For not taken action code is 0*/
+        intentNotTaken.setAction("0");
+        PendingIntent pendingIntentNotTaken = PendingIntent.getBroadcast(context, 0, intentNotTaken, PendingIntent.FLAG_UPDATE_CURRENT);
+        NotificationCompat.Action actionNotTaken;
+        actionNotTaken = new NotificationCompat.Action(R.drawable.ic_cancel_black_18dp, context.getString(R.string.drug_reminder_notification_action_not_taken)
+                , pendingIntentNotTaken);
+
+         /*Action  Taken*/
+        Intent intentTaken = new Intent(context, DrugReminderReceiver.class);
+       /* For  taken action is 2*/
+        intentTaken.setAction("2");
+        PendingIntent pendingIntentTaken = PendingIntent.getBroadcast(context, 0, intentTaken, PendingIntent.FLAG_UPDATE_CURRENT);
+        NotificationCompat.Action actionTaken;
+        actionTaken = new NotificationCompat.Action(R.drawable.ic_done_black_18dp, context.getString(R.string.drug_reminder_notification_action_taken)
+                , pendingIntentTaken);
+
+
+        Uri sound = Uri.parse("android.resource://" + context.getPackageName() + "/" + R.raw.soundsmedication);
+        /**Building Notifications**/
+        NotificationCompat.Builder builder = new NotificationCompat.Builder(
+                context).setContentTitle(context.getString(R.string.drug_reminder_notification_title))
+                .setSmallIcon(R.drawable.appicon_themed)
+                .setStyle(new NotificationCompat.BigTextStyle().bigText(context.getString(R.string.drug_reminder_notification_message)))
+                .setContentText(context.getString(R.string.drug_reminder_notification_message))
+                .setAutoCancel(false)
+                .setOngoing(true)
+                .addAction(actionNotTaken)
+                .addAction(actionTaken);
+
+        if (addSnoozeButton()) {
+              /*Action Snooze*/
+            Intent intentSnooze = new Intent(context, DrugReminderReceiver.class);
+            /* For snooze action code is 1*/
+            intentSnooze.setAction("1");
+            PendingIntent pendingIntentSnooze = PendingIntent.getBroadcast(context, 0, intentSnooze, PendingIntent.FLAG_UPDATE_CURRENT);
+            NotificationCompat.Action actionSnooze;
+            actionSnooze = new NotificationCompat.Action(R.drawable.ic_snooze_black_18dp, context.getString(R.string.drug_reminder_notification_action_snooze)
+                    , pendingIntentSnooze);
+
+            builder.addAction(actionSnooze);
+        }
+
+
```

Comment:
Remove extra lines throughout the code. A gap of one line is enough between two code segments.